src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public Ball getRandomBall() {
+        String color = getRandomColor();
+        int number = new Random().nextInt(100) + 1;
```

Comment:
random should be a class-level variable, it will belong to the class instance. Right now it's a local variable that is recreated per each method call
100  - class-level constant (don't forget CONSTANT_CASE for naming)